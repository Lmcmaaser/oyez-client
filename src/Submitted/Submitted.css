/*mobile view*/
.instructions {
  display: flex;
  flex-direction: column;
  margin: 10px;
}

.info {
  display: flex;
  flex-direction: column;
  /* align-items: center; */
  justify-content: center;
  padding: 5px;
  margin: 0px;
  background-color: #B0AFE3;
  border: 5px solid #B0AFE3;
  border-radius: 8px;
}

/*desktop view*/
@media screen and (min-width: 600px) {
  .instructions {
    display: flex;
    flex-direction: column;
    margin: 10px;
  }

  .info {
    display: flex;
    flex-direction: column;
    /* align-items: center; */
    justify-content: center;
    padding: 5px;
    margin: 0px;
    background-color: #B0AFE3;
    border: 5px solid #B0AFE3;
    border-radius: 8px;
  }
}
