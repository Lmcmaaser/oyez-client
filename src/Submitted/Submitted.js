import React from 'react';

export default class Submitted extends React.Component {
  render () {
    return (
      <div>
        <div className="instructions">
          <p className="info">Done! Your report was sent perfectly. </p>
          /* something about having your other household get tested or fill out report */
        </div>
      </div>
    )
  }
}
