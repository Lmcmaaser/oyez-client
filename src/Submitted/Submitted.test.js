import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Submitted from './Submitted';

it('renders a .instructions by default', () => {
  const wrapper = shallow(<Submitted />);
  expect(toJson(wrapper)).toMatchSnapshot();
});

it('renders without crashing', () => {
  shallow(<Submitted />);
});
