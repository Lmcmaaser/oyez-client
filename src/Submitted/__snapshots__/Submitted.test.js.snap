// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a .instructions by default 1`] = `
<div>
  <div
    className="instructions"
  >
    <p
      className="info"
    >
      Done! Your report was sent perfectly. 
    </p>
  </div>
</div>
`;
